website/docs/r/compute_backend_service.html.markdown

```diff
@@ -133,3 +133,11 @@ exported:
 * `fingerprint` - The fingerprint of the backend service.
 
 * `self_link` - The URI of the created resource.
+
+## Import
+
+Backend services can be imported using the `name`, e.g.
+
+```
+$ terraform import google_compute_backend_service.website my_backend_service
```

Comment:
The names are pretty bad though, 'foobar', and 'blablah'. If I was reading the bottom example I might get confused as to what's what.

How about I update the example to match the import instead?